package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "webpack.config.js",
   "scripts": {
-    "dev": "webpack-dev-server",
-    "cljs": "shadow-cljs --dev",
+    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js",
+    "cljs": "./node_modules/shadow-cljs/bin/shadow-cljs --dev",
```

Comment:
Based on experience I think this is unnecessary. Here's the docs I found:

https://docs.npmjs.com/misc/scripts#path

```
=>> npm run path

> minimal-cljs-webpack-hmr@0.1.0 path /Users/chen/repo/minimal-xyz/minimal-cljs-webpack-hmr
> echo $PATH

/usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/chen/repo/minimal-xyz/minimal-cljs-webpack-hmr/node_modules/.bin:/Users/chen/.yarn/bin:/Users/chen/.autojump/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/chen/.nimble/bin
```